### PR TITLE
DOC Document new community roles

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue
- https://github.com/silverstripe/.github/issues/142

### CODE REVIEW

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (michalkleiner) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
* If you don't feel confident clicking merge, ask for a second opinion. Don’t approve the pull request if you wouldn’t feel confident merging it - instead, say where you got to with it (e.g. tested, worked well locally, couldn’t find any problems but want another opinion) and get another reviewer to check it
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (michalkleiner) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: Swap around the points. Talking about folks being rude as the first escalation point feels like going into this already with a pre-negative mindset.

```suggestion
* Escalate pull requests to Core Committers if the PR is very complex and you don't feel confident reviewing it
  * If another reviewer feels confident, they can offer to review it instead
* Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
  * If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
```

Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done (I'm assuming there were no other edits in that suggestion - GitHub is bad at showing suggested changes that just swap items around)

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:24]: I would stick that in the first section or under a "Scope" header.

```suggestion
The maintainer guidelines apply to [supported modules](./supported_modules/). Unsupported modules in the `silverstripe` GitHub organisation are not necessarily maintained by these roles. If there is an unsupported module in the `silverstripe` GitHub organisation that you would like to maintain, please email <community@silverstripe.org>.
```

Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: I would stick that in the first section or under a "Scope" header.

```suggestion
The maintainer guidelines apply to [supported modules](./supported_modules/). Unsupported modules in the `silverstripe` GitHub organisation are not necessarily maintained by these roles. If there is an unsupported module in the `silverstripe` GitHub organisation that you would like to maintain, please email <community@silverstripe.org>.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: This is a web writing edit.

[Front loading your links](https://www.w3.org/WAI/wcag-curric/sam110-0.htm) makes your web content more readable. You should also make sure that your link text make sense without needing to be contextualise by the surrounding text.

```suggestion
The [maintainer house rules](#house-rules) must be followed by all maintainers.
```

I'm not going to make a big fuss about this

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
The people looking after the Silverstripe CMS Core modules.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:56]: Can we create a card to review these notice down the track?

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: I think "ability" here is better. 

```suggestion
This role is made up of community members who have demonstrated an ability to improve Silverstripe CMS through their contributions to supported modules. It empowers community members to have a more active role in ensuring pull requests get merged, and builds more momentum for community-created pull requests.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:61]: This section seems to apply to all 3 "maintainer" roles. Maybe part of this paragraph should be directly after the "Maintainer Roles" header?

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: Looks like we are trying to establish a correspondance between core committers, CMS Squad and peer reviewers. If that's the intention, then maybe we should make this explicit with a blurb like this:

> They are 3 types of maintainer roles:
> - Core committers
> - CMS Squad members
> -  Peer reviewers.

Maybe we want a table to quickly show what each type can and can't do?

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: Are we saying that we need a 3 person to merge the PR here? Could the person doing the merging be one of the two peer reviewer?

Do we need to clarify what "understand and approve the changes" means here. If 2 senior contributors did an in-depth review of a change and tested it, I wouldn't want to need a third person to replicate their work.

I would expect that 3rd person would do something like:
- validate that the first two peer reviewers did their job as expected
- do passing analysis to make sure there's no substantial concerns that went under the radar, without doing a full review.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:78]: Do we need to clarify what happens if 2+ reviewers approve of a PR, but another rejects it and won't budge?

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
If a reviewer doesn’t review any pull requests in a 12 month period, a member of the Core Committers or the CMS Squad team leader will reach out and ask if they want to continue being a reviewer. If they don’t respond within a month (or they respond saying they don’t want to be a reviewer anymore), their permissions will be revoked and they will be removed from the team.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]: 2 years to sit on your hands before we boot you off sound like a long time.
```suggestion
If a reviewer fails to contribute for 18 months in a row, they will be removed from the team until such a time as they can commit to performing the responsibilities of the role.
```

Maybe we need to set up a side process to make sure those things are reviewed regularly ... feed it into the quarterly access review.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
Members of the community who:
- have shown an interest in helping to maintain Silverstripe CMS
- have shown themselves to be trustworthy
- follow the code of conduct and 
- have provided good contributions may be invited by a Core Committer or the CMS Squad team leader to join the peer review team.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: I might review this section to mention the presence of the CMS Squad team lead.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
Those contributions can be:
- good quality bug reports
- good quality pull requests
- help refining issues and PRs
- unofficial pull request reviews.

These contributions must demonstrate clear communication, adherence to our contribution guidelines, and good collaboration skills.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md:104]: I would add something about a basic background check. e.g.: Validate that the person actually exist and is who they say they are.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
This role is made up of community members who have demonstrated an ability to improve Silverstripe CMS through their contributions to the open source supported modules. It empowers community members to have a more active role in identifying and refining bug reports and pull requests with high value potential so that they can be resolved more quickly.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
Having a role for refining issues and pull requests helps to ensure:
- High impact and impact high value contribution are prioritise.
- Contributions are in a good state by the time someone comes to work on the issue or review the pull request.
- Low quality contributions are discarded.

It also gives community members a low-effort entry point to being involved outside of contributing code themselves.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
Members of the community may be invited by a Core Committer or the CMS Squad team leader to join the contribution refiners team if they:
- have shown an interest in helping to maintain Silverstripe CMS
- have shown themselves to be trustworthy
- follow the code of conduct
- have provided good contributions.
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [inline en/06_Project_Governance/03_Maintainer_Guidelines.md]:
```suggestion
Those contributions could be:
- good quality bug reports
- good quality pull requests
- help refining issues and PRs
- unofficial pull request reviews. 

These contributions must demonstrate clear communication, adherence to our contribution guidelines, and good collaboration skills.
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Looks good.

Is this target to 4.13 on purpose?

I'm thinking maybe we need to have a formal signed agreement before granting people access. But that can maybe be sorted out later.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: > This is a web writing edit.

What does this mean?

> I'm not going to make a big fuss about this

Do you want me to make this change or not? I'm confused about this comment.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:56]: I'm happy for any cards to be created that you feel like need to be created. It feels like you have a specific idea in mind for the card so it would probably be better for you to create it. A card I create based on your suggestion would likely miss a lot of context that you have in your head.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: > I think "ability" here is better.

Can you please explain your reasoning? I think "willingness" here is better because that's the main attribute that this paragraph is talking about people having. "ability" is discussed in the section about onboarding.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: > Maybe we need to set up a side process to make sure those things are reviewed regularly ... feed it into the quarterly access review.

There's no maybe. We do need to do that.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:61]: I've put this here because it's explicitly discussing why _this_ role exists. Also there's 4 maintainer roles.

- CMS Squad doesn't exist to review pull requests, they exist to do dev work primarily. We're also _not_ members of the community for the purposes of that last sentence.
- Core Committers don't exist to review pull requests really - that's why we need this new role.
- Refiners don't get given the authority that this is talking about - their responsibilities do not include peer review per se.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: There are four types of maintainer roles, including refiners below.

I don't think that list or table is necessary, since the headings already serve that purpose.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: > Are we saying that we need a 3 person to merge the PR here? Could the person doing the merging be one of the two peer reviewer?

This is explained in the very next line: "The person who contributed the PR must not be included as one of the two approving reviewers"

In other words:

Three people must be involved in the pull request, which include:
1. The person who creates the PR
2. A first person to approve the PR
3. A second person to approve the PR and then merge it

If the CMS Squad and Core Committers are merging, then only two people are required:
1. The person who creates the PR
2. The CMS Squad member or Core Committer who approves and merges the PR

Just as the CMS Squad and Core Committers are discourages from self-merging, peer reviewers shouldn't self-approve and self-merge.

> Do we need to clarify what "understand and approve the changes" means here.

I don't think so? "understand" is hopefully self-explanatory. "Approve" means you click the "approve" button. That doesn't seem ambiguous but if you think it is, please provide a specific suggestion and explain why it makes it clearer.

> If 2 senior contributors did an in-depth review of a change and tested it, I wouldn't want to need a third person to replicate their work.
>
> I would expect that 3rd person would do something like:
> - validate that the first two peer reviewers did their job as expected
> - do passing analysis to make sure there's no substantial concerns that went under the radar, without doing a full review.

The third person doesn't need to replicate anything. But if the peer reviewers aren't confident about the change, they shouldn't merge it. If they want a third opinion, they should get a third opinion. We don't need to dictate what level of involvement that third opinionator should have - it will vary depending on the level of confidence of the first two reviewers, the complexity of the PR, and other factors.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:78]: I'll add a note about escalating if there are disagreements between reviewers.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: I'm not sure what this comment is for... is this a note for yourself to review this again? Or are you requesting a change? Or...?

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:104]: It seems implied but I'll add it explicitly.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Discussion about this will be in https://github.com/silverstripe/developer-docs/pull/394#discussion_r1403903814

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done, but now that it's talking about the guidelines and not the roles I've moved it to be below the guidelines heading. I've also moved something that _was_ under that heading but talks about the roles, and put that under the roles heading.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:78]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: I had 24 months because that means we only ever have to do a single review every 12 months. This change requires a review every 6 months, but the timeline is more sensible. I'll make the change.

Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done, except I moved the"may be invited by a Core Committer or the CMS Squad team leader to join the peer review team" part to be before the list instead of dangling on the last bullet point.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:104]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: > High impact and impact high value contribution are prioritise.

I assume this was meant to be "high impact and high value contributions are prioritised"... which is kiiiiind of part of this role, so I'll add it.

Done.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: You're really all in on lists for this page 😂
Done.

**Comment by GuySartorelli** [comment]
> Is this target to 4.13 on purpose?

Yes. Anything that affects both CMS 4 and 5 _or_ which doesn't apply to a specific version should target `4.13` first and be merged up from there.

Ideally these pages wouldn't be stuck behind a version toggle - see https://github.com/silverstripe/doc.silverstripe.org/issues/245
But since they are for now, this is the process we must follow to avoid the problem getting worse than it already is.

> I'm thinking maybe we need to have a formal signed agreement before granting people access. But that can maybe be sorted out later.

Yup, we can update the docs if/when we have such an agreement.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: I meant something like this for the table.

| Responsibilities / Powers                               | Contribution Refiners | Peer Reviewers | CMS Squad members | Core committers |
|---------------------------------------------------------|-----------------------|----------------|-------------------|-----------------|
| Triage issue                                            | x                     | x              | x                 | x               |
| Review pull requests                                    | x                     | x              | x                 | x               |
| PR approval counts towards merge total                  |                       | x              | x                 | x               |
| Merge PRs without meeting minimal approval requirements |                       |                | x                 | x               |
| Administer repositories                                 |                       |                |                   | x               |

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Yes, we should make that change.

I was just detailing it a bit more here because we've been absolutely horrible when drafting content and we have link splatter everywhere.

As a general principal, we should be following these guidelines when drafting docs.

The [NZ Digital guidelines on links](https://www.digital.govt.nz/standards-and-guidance/design-and-ux/content-design-guidance/links/) are pretty good and are reflective of the web writing consensus.

> Guidelines
> 
> The Readability Guidelines include detailed guidelines on the following topics:
> 
> - Make link text meaningful.
> - Avoid mid-sentence links.
> - Front-load your link text.
> - Make call to action (CTA) links and button text specific.
> - Start CTA links and button text with a verb.
> - Make CTA links and button text 2 to 4 words.
> - Match the destination content.
> - Use sentence case.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: We reference the CMS Squad team lead through out the document without mentioning where they fit in the grander scheme of things or what they do.

I'm just thinking it might be worthwhile adding a bit more context here.

> The CMS Squad is managed by the CMS Squad team lead. The CMS Squad team lead acts as product owner for Silverstripe CMS and is the primary point of contact between the Core Committers and Silverstripe.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: If someone was willing, but we didn't think they are able to do the job correctly, we wouldn't appoint them. I would focus more on ability.

We could go with "have demonstrated a willingness and ability to improve Silverstripe CMS" if we think both attribute are equally important.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: That makes sense. My main concern is that the context of "The person who merges the PR must also understand and approve the changes" bullet point gives the impression that "the person who merges the PR" needs to be someone other than two maintainers who approved it.

Maybe add another bullet point to make it explicit that the approving maintainers can merge the PRs they have reviewied:
```suggestion
    * Approving maintainers can merge PRs they have reviewed
    * The person who merges the PR must also understand and approve the changes
```

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: [List are more readable](https://www.digital.govt.nz/standards-and-guidance/design-and-ux/content-design-guidance/content-structure/lists/). Generally, whenever something can be broken into a list, it should.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Almost there.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Yup, fair. It wasn't a critique, just an amused observation.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done - any sweeping changes you want can be raised as a separate card.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: I really don't see a need for it but I'll add it to avoid peer review ping pong

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done, but with different wording

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: Done, but with "leader" instead of "lead"

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md]: Done as per the linked comment

**Review by GuySartorelli — COMMENTED** [review]
Note: The branch this PR is targetting now has a CI action. This PR will need to be force-pushed to run the CI, and the CI must be green before merging.

**Comment by GuySartorelli** [comment]
Rebased and resolved conflicts, lint passes locally.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: My official job title says "Lead".

There could be a case for calling this something else (e.g.: Silverstripe CMS Project Owner). If I drop dead tomorrow, it's more likely than not that the CMS Squad would end up with a Delivery Lead. But Silverstripe CMS will always have a PO, no matter what their title is.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (maxime-rainville) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:56]: https://github.com/silverstripe/developer-docs/issues/414

**Review by maxime-rainville — COMMENTED** [review]
The only thing left is a pedantic and egocentric argument about my Job title.

File: en/06_Project_Governance/03_Maintainer_Guidelines.md
Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: I really don't think this document cares about job titles. I'll update mentions of the "CMS squad team leader" (i.e. leader of the CMS squad) to mention instead the "Silverstripe CMS Product Owner" since that's really the relevant role for those mentions.

Comment (ChloeSartorelli) [reply en/06_Project_Governance/03_Maintainer_Guidelines.md:50]: Done. Note I have also removed the note about the CMS Squad being managed by the team leader as that's no longer relevant.

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**
